pkgs/os-specific/linux/nixos-rebuild/_nixos-rebuild

```diff
@@ -71,7 +72,14 @@ _nixos-rebuild() {
     --keep-going -k
     --max-jobs -j # number
     --log-format # format
+    --quiet
+    --print-build-logs -L
+    --no-build-output -Q
+    --fallback
+    --refresh
+    --repair
     -I # NIX_PATH
+    --offline # --no-net not suggested, deprecated
```

Comment:
`--no=net` is preferred over `offline` nowadays?

Reply:
Hm? I don't understand.

`--no-net` is a now deprecated `nix` argument. So I'm not adding it to the bash completion. Does that help?

Reply:
ah saw your comment below sorry for noise